kaldi/install.bash

```diff
@@ -14,10 +14,14 @@ prev="-1"
 # TODO: Change this to tue-install-targets --now $name_of_targets
 tue-install-system-now zlib1g-dev automake autoconf patch grep \
     bzip2 gzip wget sox libtool subversion gawk python python3 libatlas3-base \
-    swig zip p7zip-full python-six python-numpy gstreamer1.0-pulseaudio \
+    swig zip p7zip-full python-six python-numpy libglib2.0-dev \
+    libgstreamer1.0-dev gstreamer1.0-tools gstreamer1.0-pulseaudio \
     gstreamer1.0-plugins-base gstreamer1.0-plugins-bad \
     gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly
 
+# The latest version of numpy is required
+pip install --user -U numpy
```

Comment:
Please define a minimum version here. As 'the latest' isn't valid in the future.